goth/runner/cli/yagna_payment_cmd.py

```diff
@@ -72,19 +68,17 @@ def from_dict(source: dict):
 class YagnaPaymentMixin:
     """A mixin class that adds support for `<yagna-cmd> payment` commands."""
 
-    def payment_fund(
-        self: CommandRunner, payment_driver: PaymentDriver = DEFAULT_PAYMENT_DRIVER
-    ) -> None:
+    def payment_fund(self: CommandRunner, payment_driver: str) -> None:
         """Run `<cmd> payment fund` with optional extra args."""
-        args = make_args("payment", "fund", driver=payment_driver.name)
+        args = make_args("payment", "fund", driver=payment_driver)
         self.run_command(*args)
 
     def payment_init(
```

Comment:
Same as above, maybe let's use a `PaymentConfig` object here? In this case we could use it instead of `payment_driver` and `network` parameters.